packages/console/src/hooks/use-scroll.ts

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect, useCallback } from 'react';
+
+const useScroll = (ref?: HTMLDivElement) => {
```

Comment:
nit: can remove `?`, make it required

Reply:
No, we can't.
The `ref.current` from `useRef` is optional.
And we cannot use `RefObj<HTMLDivElement>` as its parameter, for we will lose track of the reference when it is mounted.

Reply:
per offline discussion, updated with `Nullable`.